src/converters/xml.ts

```diff
@@ -87,8 +125,14 @@ export function gpx(doc: string | Document): GeoJSON.FeatureCollection {
 export function kml(
   doc: string | Document,
   xml2string?: (node: Node) => string,
-): GeoJSON.FeatureCollection {
+): GeoJSON.FeatureCollection | null {
```

Comment:
```suggestion
): GeoJSON.FeatureCollection {
```

Throwing an error will not return a null but rather control flow is transferred to the nearest enclosing `try`...`catch` block.